pkg/utils/switchdev.go

```diff
@@ -11,9 +12,20 @@ import (
 )
 
 const (
-	switchDevConfPath = "/host/etc/switchdev.conf"
+	switchDevConfPath = "/host/etc/sriov_interface_config.json"
 )
 
+type config struct {
+	Interfaces []Interface `json:"interfaces"`
+}
+
+type Interface struct {
```

Comment:
@pliurh  can we use the interface structure in sriovnetworknodestate API?

Reply:
Good idea. Although we don't support configuring all the flags now, we might support them in the future.